.github/ISSUE_TEMPLATE

```diff
@@ -4,25 +4,14 @@ Provide a description of what is the current problem and why you are raising thi
 If it's a bug please describe what was the unexpected thing that occured and what was the
 expected behaviour.
 
-## Solution
-
-Here provide a detailed explanation, if you can suggest one, for how to go about solving the problem.
-
-
-## Tasklist
```

Comment:
As we discussed the tasklist and solution approach should be for our own internal issues and not the bugs, so perhaps move it to the contributing guide?

Reply:
sure

Reply:
Perhaps the creation of a task list with the tasks that the issue creator thinks needs to be completed before the issue is considered complete.

Reply:
@hackaugusto I will merge this PR now and either you or me can add the deleted part from the issue template to the guide. Reason is: We need it now. Yet again, someone mis-used the template to create their issue and it gets on my nerves.